docs/features/nullable-reference-types.md

```diff
@@ -157,9 +163,16 @@ var y = (IEnumerable<object>)x;  // warning
 var z = (IEnumerable<object?>)x; // no warning
 ```
 
+### Method type inference
+_Describe details_
```

Comment:
Why were the details removed?

Reply:
@gafter has a spec. I've logged #30433 to add that here.

---
In reply to: [224233359](https://github.com/dotnet/roslyn/pull/29727#discussion_r224233359) [](ancestors = 224233359)